diff --git a/wireguard-src/src/crypto/chacha20poly1305.c b/wireguard-src/src/crypto/chacha20poly1305.c
index c291ba9..10ab915 100644
--- a/wireguard-src/src/crypto/chacha20poly1305.c
+++ b/wireguard-src/src/crypto/chacha20poly1305.c
@@ -39,7 +39,7 @@ void chacha20poly1305_init(void)
 {
 	chacha20poly1305_use_sse2 = boot_cpu_has(X86_FEATURE_XMM2);
 	chacha20poly1305_use_ssse3 = boot_cpu_has(X86_FEATURE_SSSE3);
-	chacha20poly1305_use_avx2 = boot_cpu_has(X86_FEATURE_AVX2);
+	chacha20poly1305_use_avx2 = false;
 }
 #else
 void chacha20poly1305_init(void) { }
